server/controllers/tournamentController.ts

```diff
@@ -28,9 +28,14 @@ const updateMatchOutcomes = catchAsync(
 );
 
 const getAllTournaments = catchAsync(async (req, res) => {
-  const tournaments = await TournamentRepository.getAll();
+  const page = Number(req.query.page) || 1;
+  const pageSize = Number(req.query.pageSize) || 100;
+  const tournaments = await TournamentRepository.getAll(page, pageSize);
```

Comment:
hmm what will happen when when i input `page: -50` or `pageSize: -100` or `page: 'XYZ'`? Do we need validation?

Reply:
Mongoose will reject